.github/workflows/snap.yaml

```diff
@@ -34,7 +34,6 @@ jobs:
     secrets:
       store-login: ${{ secrets.STORE_LOGIN }}
     with:
-      branch-name: feature/multi-snap
```

Comment:
removing this will break the monthly build, with the new modification this PR should include the

with:
  branch-name: feature/multi-snap
  
  in the monthly.yaml file

Reply:
Open a follow-up at #38